docker/README.md

```
@@ -93,6 +93,7 @@ regarding FiftyOne Enterprise.
     can then use utilization metrics to make scaling decisions (up or down).
     Please note that we do not support MongoDB Atlas Serverless instances
     because we require Aggregations.
+  - A FiftyOne Enterprise deployment requires two separate databases in your MongoDB instance that are automatically created on startup (defaulting to [`fiftyone`](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/17c11828f3e3d02cc7eb32543d26ba3e61ebfe77/docker/internal-auth/env.template#L52) and [`fiftyone-cas`](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/17c11828f3e3d02cc7eb32543d26ba3e61ebfe77/docker/internal-auth/env.template#L64))
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 461

(MD013, line-length)